SH4RE/SH4RE/ViewListing/ViewListing.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  ViewListing.swift
+//  SH4RE
+//
+//  Created by Hannah Brooks on 2022-11-24.
+//
+
+import SwiftUI
+import Firebase
+
+class ViewListing: ObservableObject {
+
+    var test_listing = Listing(id :"MNizNurWGrjm1sXNpl15", title:"Test Listing", description: "Test Description", imagepath : "path", price: "$20.00")
```

Comment:
not sure if you were testing with this but the $ needed to be removed from dis tester bc that could change some things but probably not since viewlistingview is always called from an actually listing so maybe nvm. But this file may not actually be needed since it isn't technically its own page

Reply:
no i did not. i can take it out. left from bryans testing.